.github/workflows/release.yaml

```diff
@@ -52,96 +52,33 @@ jobs:
         with:
           name: release-ubuntu-18.04
           path: release/ubuntu-18.04
-      - name: Delete artifact
-        uses: geekyeggo/delete-artifact@v1
-        with:
-          name: release-ubuntu-18.04
-          failOnError: false
```

Comment:
Why were the delete-artifact steps removed?

Reply:
Because we're using the `--clobber` option to `gh release upload`. I assumed the only purpose of the delete was to allow the later upload to work.

Reply:
No, these were used to delete the run artifacts created by upload-artifact@v2 because they are stored for 90 days and take up some storage quota

Reply:
Okay, I restored those. Please take another look.